flytekit/remote/nodes.py

```diff
@@ -50,15 +43,20 @@ def __init__(
         elif flyte_launch_plan is not None:
             workflow_node = _component_nodes.FlyteWorkflowNode(flyte_launch_plan=flyte_launch_plan)
 
+        task_node = None
+        if flyte_task:
+            task_node = _component_nodes.FlyteTaskNode(flyte_task)
+        branch_node = None
+
         super(FlyteNode, self).__init__(
-            id=_dnsify(id) if id else None,
```

Comment:
why we don't need to `_dnsify` anymore?

Reply:
a lot of this code was copied from the SdkXyz classes, which were used on both read and write.  remove cuz before the id came from user input basically, now we're reading from admin, so the id should be the id, we shouldn't mess with it.